conf/messages.cy

```diff
@@ -469,7 +471,9 @@ technical.issue.heading=Mae’n ddrwg gennym, mae problem gyda’r gwasanaeth
 technical.issue.try.later=Rhowch gynnig arall arni yn nes ymlaen.
 technical.issue.back=Dychwelyd i GOV.UK
 technical.other-ways.h1=Ffyrdd eraill o wneud cais am Lwfans Priodasol
-technical.other-ways.para0=Gallwch wneud cais am Lwfans Priodasol drwy''ch <a href="https://www.gov.uk/cyfrif-treth-personol">Cyfrif Treth Personol</a>, o dan ‘gwasanaethau y mae''n bosibl y bydd eu hangen arnoch’.
+technical.other-ways.para0.part1=Gallwch wneud cais am Lwfans Priodasol drwy''ch
+technical.other-ways.para0.link-text=Cyfrif Treth Personol
+technical.other-ways.para0.part2=, o dan ‘gwasanaethau y mae''n bosibl y bydd eu hangen arnoch’.
```

Comment:
Wrong type of quotes

Reply:
These quotes are from the original message. Which ones are the right ones?

Reply:
Smart quotes in place of the double single quote ''